roles/matrix-mxisd/defaults/main.yml

```diff
@@ -43,6 +43,16 @@ matrix_mxisd_dns_overwrite_enabled: false
 matrix_mxisd_dns_overwrite_homeserver_client_name: "{{ matrix_server_fqn_matrix }}"
 matrix_mxisd_dns_overwrite_homeserver_client_value: "http://matrix-synapse:8008"
 
+# Override the default email templates
```

Comment:
Maybe we should have a comment that says that these variables are meant to contain actual template contents (and not a file-path)?

Reading the actual task logic makes it clear, but only looking at the variables may be confusing.

Reply:
Agreed and changed